HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -165,6 +165,13 @@ def customiseFor18429(process):
 
      return process
 
+# Updating FatjetJetProducer with fillDescriptions
+def customiseFor18832(process):
+    for producer in producers_by_type(process, "FastjetJetProducer"):
+        if not hasattr(producer, "doOutputJets"): producer.doOutputJets = cms.bool(False)
+        if not hasattr(producer, "sumRecHits"): producer.sumRecHits = cms.bool(False)
```

Comment:
Did you try this customization? I think you must do the contrary, instead: if there is that parameter in the config, you must remove it. The syntax should be

     if hasattr(producer, "doOutputJets"): del producer.doOutputJets
     if hasattr(producer, "sumRecHits"): del producer.sumRecHits

But please check yourself if it works before re-committing

@Martin-Grunewald